api/helpers/auth.js

```diff
@@ -1,11 +1,9 @@
 'use strict';
 
-var jwt = require('jsonwebtoken');
+const jwt = require('jsonwebtoken');
 const jwksClient = require('jwks-rsa');
-
-// Old keycloak endpoints
-var ISSUER = process.env.SSO_ISSUER || "https://oidc.gov.bc.ca/auth/realms/aaoozhcp";
-var JWKSURI = process.env.SSO_JWKSURI || "https://oidc.gov.bc.ca/auth/realms/aaoozhcp/protocol/openid-connect/certs";
+var ISSUER = process.env.SSO_ISSUER || "https://dev.loginproxy.gov.bc.ca/auth/realms/standard";
+var JWKSURI = process.env.SSO_JWKSURI || "https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs";
```

Comment:
Why is this hardcoded, this url is different for each env.  Did i miss something here, or does this use a variable somewhere else.  If so great

Reply:
Yes, we'll need to update our DeploymentConfigs to have their own SSO_JWKSURI env vars. The fallback is used locally when none is set.